README.md

```diff
@@ -58,20 +58,30 @@ yarn coverage
 
 ## Deploy & verify
 
-### Setup
+### Environment Setup
 
-Configure the `.env` variables.
+Copy the sample environment file located in `env/` into the target subdirectory of your choice (e.g., `testnet` or `mainnet`) and prefix the filename with your blockchain of choice:
 
-### Sepolia
-
-```bash
-yarn deploy:sepolia
+```
+mkdir env/testnet
+cp env/.env.sample env/testnet/sepolia.env
```

Comment:
we should create the env files in these directories, and example afterwards. this way it would be to easy to copy paste

Reply:
wdym by this?

Reply:
Instead of creating the env file sample at env/

Create it as env/testnet/sepolia.env.sample
This way it would be easier to kickstart with it.

Reply:
env.sample is a standard file for every environment
sepolia is just an example, for mainnet ethereum it'd be env/mainnet/ethereum.env
and similar for polygon amoy env/testnet/amoy.env

I don't understand, why would we create sample for all networks

Reply:
not for all the network just single file for testnet and mainnet. Cause right now the dir won't be created for each which means. Anyone onboarding will have to create dir and then add files for the network.